keywords/LiteServiOS.py

```diff
@@ -74,11 +71,14 @@ def install_device(self):
         Warning: Only works with a single device at the moment
         """
 
-        if self.storage_engine != "SQLite":
-            raise LiteServError("https://github.com/couchbaselabs/liteserv-ios/issues/1")
+        if self.storage_engine == "SQLCipher":
+            package_name = "LiteServ-iOS-SQLCipher-Device.app"
```

Comment:
It is there, it is added with my request by build team and Pasin. When I was having the support for iOS device , I do not see app for device, that is the reason there was no support for Device. Now Pasin and build team added, so that we can run it on device